papers/jupyter_book/myst.yml

```
@@ -0,0 +1,369 @@
+        - dra
+      roles:
+        - Software
+        - Investigation
+        - Writing – review & editing
+    - id: chris
+      name: Chris Holdgraf
+      email: choldgraf@2i2c.org
+      orcid: 0000-0002-2391-0678
+      github: choldgraf
+      affiliations:
+        - id: 2i2c
+        - id: jupyter
+      roles:
+        - Conceptualization
+        - Funding acquisition
+        - Investigation
+        - Project administration
+        - Software
+        - Supervision
+        - Writing – original draft
+        - Writing – review & editing
+    - id: angus
+      name: Angus Hollands
+      email: ahollands@2i2c.org
+      orcid: 0000-0003-0788-3814
+      affiliations:
+        - id: 2i2c
+        - id: jupyter
+      roles:
+        - Investigation
+        - Software
+        - Writing – review & editing
+    - id: julia
+      name: Julia Kent
+      email: jkent@ucar.edu
+      orcid: 0000-0002-5611-8986
+      github: jukent
+      affiliations:
+        - ncar
+      roles:
+        - Writing – review & editing
+    - id: franklin
+      name: Franklin Koch
+      orcid: 0000-0002-6393-7058
+      github: fwkoch
+      affiliation:
+        - id: curvenote
+        - id: jupyter
+      roles:
+        - Investigation
+        - Software
+        - Writing – review & editing
+    - id: jim
+      name: Jim Madge
+      email: jim@jmadge.com
+      orcid: 0000-0001-6044-164X
+      github: JimMadge
+      affiliations:
+        - alanturinginstitute
+        - ttw
+      roles:
+        - Investigation
+        - Software
+        - Writing – review & editing
+    - id: matt
+      name: Matthew McKay
+      email: mamckay@gmail.com
+      github: mmcky
+      affiliations:
+        - quantecon
+      roles:
+        - Conceptualization
+        - Funding acquisition
+        - Project administration
+        - Supervision
+        - Writing – review & editing
+    - id: mike
+      name: Mike Morrison
+      email: mike@curvenote.com
+      orcid: 0000-0002-1324-8801
+      affiliation:
+        - curvenote
+      roles:
+        - Validation
+        - Writing – review & editing
+    - id: fernando
+      name: Fernando Pérez
+      orcid: 0000-0002-1725-9815
+      email: fernando.perez@berkeley.edu
+      affiliations:
+        - id: berkeley-stats
+        - id: berkeley-bids
+        - id: berkeley-dse
+        - id: lbl
+        - id: jupyter
+      roles:
+        - Writing – review & editing
+    - id: steve
+      name: Steve Purves
+      email: steve@curvenote.com
+      orcid: 0000-0002-0760-5497
+      github: stevejpurves
+      affiliations:
+        - id: jupyter
+        - id: curvenote
+        - id: csf
+      roles:
+        - Conceptualization
+        - Funding acquisition
+        - Investigation
+        - Project administration
+        - Software
+        - Supervision
+        - Writing – review & editing
+    - id: min
+      name: Min Ragan Kelley
+      email: benjaminrk@gmail.com
```

```suggestion
      email: benjaminrk@gmail.com
      orcid: 0000-0002-1023-7082
      github: minrk
```

@minrk - I noticed you were missing an ORCID - I'm pretty confident that you're ok with it being added?!

I've also just added your GitHub because I'm on a bit of a roll.....